pkg/export/otel/metrics.go

```diff
@@ -813,20 +816,34 @@ func otelHistogramConfig(metricName string, buckets []float64, useExponentialHis
 }
 
 func (mr *MetricsReporter) metricResourceAttributes(service *svc.Attrs) attribute.Set {
-	attrs := []attribute.KeyValue{
+	baseAttrs := []attribute.KeyValue{
 		request.ServiceMetric(service.UID.Name),
 		semconv.ServiceInstanceID(service.UID.Instance),
 		semconv.ServiceNamespace(service.UID.Namespace),
 		semconv.TelemetrySDKLanguageKey.String(service.SDKLanguage.String()),
 		semconv.TelemetrySDKNameKey.String("beyla"),
 		request.SourceMetric("beyla"),
+	}
+
+	extraAttrs := []attribute.KeyValue{
 		semconv.HostID(mr.hostID),
 	}
+
 	for k, v := range service.Metadata {
-		attrs = append(attrs, k.OTEL().String(v))
+		extraAttrs = append(extraAttrs, k.OTEL().String(v))
 	}
 
-	return attribute.NewSet(attrs...)
+	httpMetrics := []string{"http.server", "http.client"}
+	rpcMetrics := []string{"rpc.server", "rpc.client"}
+	dbMetrics := []string{"db.client"}
+	messagingMetrics := []string{"messaging."}
+
+	metricTypes := append(httpMetrics, rpcMetrics...)
+	metricTypes = append(metricTypes, dbMetrics...)
+	metricTypes = append(metricTypes, messagingMetrics...)
```

Comment:
as this seems to be always the same value, can't we just create a global variable?

```go
var metricTypes := []string{
   "http.server", "http.client", "rpc.server", "rpc.client" .... etc etc
```